201-aci-wordpress/azuredeploy.json

```diff
@@ -0,0 +1,247 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storageAccountType": {
+            "type": "string",
+            "defaultValue": "Standard_LRS",
+            "allowedValues": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_ZRS"
+            ],
+            "metadata": {
+                "description": "Storage Account type"
+            }
+        },
+        "storageAccountName": {
+            "type": "string",
+            "defaultValue": "[uniquestring(resourceGroup().id)]",
+            "metadata": {
+                "description": "Storage Account Name"
+            }
+        },
+        "siteName": {
+            "type": "string",
+            "defaultValue": "[uniquestring(resourceGroup().id)]",
+            "metadata": {
+                "description": "WordPress Site Name"
+            }
+        },
+        "mysqlPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "MySQL database password"
+            }
+        }
+    },
+    "variables": {
+        "image": "microsoft/azure-cli",
+        "cpuCores": "0.5",
+        "memoryInGb": "0.7",
+        "shareContainerGroupName": "[concat(resourceGroup().name, '-createshare')]",
```

Comment:
resourceGroup().name can start with non-alphnumeric chars - is that true for all these variables?

Reply:
use fixed value since they do not need to be globally unique